DrinkManagerConsole/SearchDrinkByNameConsole.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BLL;
+using System;
+using System.Collections.Generic;
+
+namespace DrinkManagerConsole
+{
+    public static class SearchDrinkByNameConsole
+    {
+        public static void StartSearch(List<Drink> drinksList)
+        {
+            bool continueSearch = true;
+            do
+            {
+                Console.Clear();
+                Console.WriteLine("\nSearch drink by name");
+                Console.WriteLine("---------------------------------------------------------------------------------------------------------------------");
+
+                // Text to search and reference to Drinks List passed into Search method - returns list
+                var drinksFound = SearchDrinkByName.SearchByName(Utility.GetGenericData("Enter a drink name to find: "), drinksList);
```

Comment:
That's kind of... odd usage of method that shows additional message in console and awaits input (I'm talking about this GetGenericData used as one of the arguments), but let's leave it for now.